FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [fixed] Return FirebaseVersion as FIRRemoteConfigPodVersion. (#7892)
```

Comment:
not sure if we need a changelog since this is only used for internal logging

Reply:
Yep, no reason for an internal change release note.